db/seeds/questions.yml

```diff
@@ -130,7 +162,41 @@ be_humble:
   question_type: long_text
   word_limit: 100
 
-# Additional Info
+# Background
+
+fa20_technical_gauge:
+  title: '[Optional] If you are applying as a developer, from the following courses, list out the ones you have taken/are currently taking:'
+  hint: "- CS 10\n- CS 61A\n- Data 8\n- CS 88\n- CS 61B\n- Atleast 1 CS upper div course (answer yes or no)\n- Other: (could include relevant decals such as the Web Design/React/Rails Decals or other learning experiences)"
+  required: false
+  question_type: short_text
+  question_category: background
+
+fa20_classes_additional_info:
+  title: '[Optional] Feel free to add anything else we should know about your answer to the above (ex. currently auditing, applied for WDD, took a class but dropped partway due to COVID).'
```

Comment:
nit: add an option for 'couldn't get in from waitlist' bc that was me first semester ;__;

Reply:
I think that's cloooose enough to just the "applying" part that it should be covered? It's awkward to phrase so can include it if you have suggestion for phrasing!